File: problems_001to025/problem_006/solution.py
import os
import sys

PROJECT_PATH = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
sys.path.append(PROJECT_PATH)

from measurement import Stopwatch
from toolbox import natural_sum, square_sum


class Solution(Stopwatch):
    def __init__(self):
        super().__init__()

    def execute(self):
        answer = natural_sum(100)**2 - square_sum(100)
        return answer

if __name__ == '__main__':
    soln = Solution()
    result = soln.execute()
    print(result)

File: problems_001to025/problem_006/solution.ts
import { SolutionFunction } from '@/lib/types';
import { naturalSum, squareSum } from '@/lib/toolbox';

const solution: SolutionFunction = () => {
  let answer: ReturnType<SolutionFunction> = null;

  answer = naturalSum(100) ** 2 - squareSum(100);

  return answer;
};

export default solution;
